docarray/array/storage/redis/getsetdel.py

```diff
@@ -95,6 +95,12 @@ def _document_to_redis(self, doc: 'Document') -> Dict:
             if tag is not None:
                 extra_columns[col] = int(tag) if isinstance(tag, bool) else tag
 
+        if self._config.tag_indices:
+            for index in self._config.tag_indices:
+                text = doc.tags.get(index)
+                if text is not None:
```

Comment:
```suggestion
                if text:
```

Reply:
I'd say keep it as it is, it can be empty string